### PR TITLE
basic control tutorial:only show first object once

### LINKED ISSUE #5336: In basic control tutorial, first objective appears twice
Please fill out the relevant sections below and delete the rest.

**Describe the bug**
In the basic control tutorial, the first objective appears two times, which is a bit confusing. It's mentioned in #4365, but it's not exactly clutter: it's duplicated information. The desired behavior would be for the welcome message->click ok->first objective.

**Screenshots**
Note the first window, and the text at the bottom:
image

After clicking "ok", the second window:

image

**Version:**
 - Widelands Version: 1.1~git50/77be5a6@master/(Debug)

**Additional context**
I spent some time trying to figure out how to fix it, but I could not. This bug is also in v1.0

### PR DESCRIPTION
**Pull request by andy5995**
(fixes #5336)(related to #4365)

image

image

image

### CODE REVIEW

**Comment by andy5995** [comment]
Oh, you know what? I did this wrong. I used "window" in two different contexts. Figures that would happen when I'm tired...

What's closer to the desired behavior?

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua [hunk 1]
```diff
@@ -39,7 +39,8 @@ obj_initial_close_objectives_window = {
    body = objective_text(_("Handling of the ‘Objectives’ window"),
       li_image("images/wui/menus/objectives.png", _([[You can open and close the ‘Objectives’ window by clicking on the ‘Objectives’ button in the toolbar on the bottom of the screen.]])) ..
       li_arrow(_([[Like most other windows, you can also close the window by right-clicking on it.]])) ..
-      li_arrow(_([[When you have accomplished an objective, it will disappear from the list.]]))
+      li_arrow(_([[When you have accomplished an objective, it will disappear from the list.]])) ..
+      li_arrow(_([[You may open the objectives window now.]]))
```
Comment (hessenfarmer) [inline data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]: you may leave out this item as it is automatically opened by  line 48 in the mission_thread

**Comment by andy5995** [comment]
k, so I've reverted the first commit, just changed very little on the second one. Here's how it looks now. Problem is that when the objective button is clicked, or if the objective dialog is right-clicked, the tutorial proceeds to the building spaces.

When the objective button is clicked, the window doesn't open, but instead the tutorial proceeds, which may confuse the user.

image

image

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua (hunk 1 above)
Comment (andy5995) [reply data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]: > you may leave out this item as it is automatically opened by line 48 in the mission_thread

Oh, *that's* what's happening... so... not really a bug then like I reported. A bit confusing but I get what's happening now.

**Review by hessenfarmer — APPROVED** [review]

**Pull request merged**